map2.js

```diff
@@ -31,3 +31,13 @@ var phoneBook = [
 // For example, the 0th element of the new array should be {Abe: "1-111-111-1111"}
 
 //Your code here
+const newArray = phoneBook.map (function(customer){
+  let name = Object.keys(customer);
+  let phoneNumber = customer[name];
+  let newRecord = {};
+  newRecord[name] = phoneNumber.charAt(0) + "-" +
+  phoneNumber;
```

Comment:
nice! not sure why you put a line break here, though? since you're not done concatenating the string, `phoneNumber` should be on the same line. Not a huge deal though, since it doesn't break the code